src/classes/ACCT_IndividualAccounts_TEST.cls

```diff
@@ -1528,4 +1528,80 @@ private class ACCT_IndividualAccounts_TEST {
         System.assertEquals('Ms. Betty, Lily and Tom White Family', requeryFinalAccount.Name);
 
     }
+
+    /*********************************************************************************************************
+    * @description In a multi-currency enabled org, create a contact with different currency than the default one.
+    * Make sure that the account automatically created and the contact have the same currency.
+    */
+    @isTest
+    private static void insertContactMultiCurrencyEnabled() {
+        if (UserInfo.isMultiCurrencyOrganization()) {
+            //Test data setup
+            String nonDefaultCurrencyCode = '';
+            String qryCurrencyTypeString = 'SELECT IsoCode FROM CurrencyType WHERE IsCorporate = false LIMIT 1';
+            List<SObject> nonDefaultCurrencies = Database.query(qryCurrencyTypeString);
+            if (nonDefaultCurrencies.size() > 0) {
+                nonDefaultCurrencyCode = (String)nonDefaultCurrencies[0].get('IsoCode');
+            } else {
+                return;
+            }
+
+            UTIL_CustomSettingsFacade.getSettingsForTests(new Hierarchy_Settings__c(
+                                                        Account_Processor__c = UTIL_Describe_API.getHhAccRecTypeID()));
+
+            Contact testContact = UTIL_UnitTestData_TEST.getContact();
+            testContact.put('CurrencyIsoCode', nonDefaultCurrencyCode);
+
+            //Run the logic
+            Test.startTest();
+            insert testContact;
+            Test.stopTest();
+
+            //Assert the result
+            String qryString = 'SELECT Id, Name, CurrencyIsoCode FROM Account';
+            List<Account> resultAccounts = Database.query(qryString);
+            system.assertEQUALS(1, resultAccounts.size(), 'More than one Account were created.');
```

Comment:
nit: System.AssertEquals() but no big deal :)

I like that you are using the assert messages here, I think we should all get in that habit. Thank you :)